tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Team: Rocket
+
+:TestType: Functional
+
+:Upstream: No
+"""
+
+import pytest
+
+
+@pytest.mark.tier2
+def test_positive_org_loc_change_for_global_registration(
+    session,
+    module_activation_key,
+    module_org,
+    module_location,
+    default_os,
+    target_sat,
+):
+    """Changing the organization and location to check if correct org and loc is updated on the global registration command
+
+    :id: e83ed6bc-ceae-4021-87fe-3ecde1cbf347
+
+    :expectedresults: organization and location is updated correctly on the global registration command
+
+    :CaseImportance: Medium
+    """
+    new_org = target_sat.api.Organization().create()
+    new_loc = target_sat.api.Location().create()
```

Comment:
as you're already using module_org and module_location, so can we skip creation of this new org and loc here?

Reply:
We also want to make sure that after changing to the new org and new loc, the global registration is also auto-updated with new org and new loc.

Reply:
ah my bad, I assumed you're using default org and loc